l10n_ch_import_camt054/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+from . import customParser
```

Comment:
file names should follow the convention with _ names. Please rename customParser to custom_parser